functions/video.js

```diff
@@ -32,6 +36,19 @@ exports.getAgoraToken = functions.https.onCall((data, context) => {
     );
   }
 
+  try {
+    assertValidAgoraConfig();
+  } catch (err) {
```

Comment:
looks like you want to use `err` in the logger few lines below. Or is there another `error` variable?
```suggestion
  } catch (error) {
```

Reply:
Nice catch!

Fixed in c0a937a6a2acba7b5cc929066d14eaabfe9d2249